static/js/dict.ts

```diff
@@ -70,15 +70,19 @@ export class Dict<K, V> {
         return dict;
     }
 
-    get(key: K): V | undefined {
+    get(key: K | string): V | undefined {
         const mapping = this._items[this._munge(key)];
         if (mapping === undefined) {
             return undefined;
         }
         return mapping.v;
     }
 
-    set(key: K, value: V): V {
+    /**
+     * `value` is set to `undefined` if not provided. This effectively makes
+     * Dict a Set.
+     */
+    set(key: K, value?: V): V {
```

Comment:
This isn’t type safe. It allows converting `undefined` to any `V`. It only compiles because we have `strictNullChecks` disabled, which we want to avoid relying on.

Reply:
Removed the commit. (Note that all of the `dict.set(key)` JS calls will still work).